libsqlite3-sys/Cargo.toml

```diff
@@ -39,7 +39,5 @@ winsqlite3 = ["min_sqlite_version_3_7_16"]
 [build-dependencies]
 bindgen = { version = "0.58", optional = true, default-features = false, features = ["runtime"] }
 pkg-config = { version = "0.3.19", optional = true }
-cc = { version = "1.0", optional = true }
-
-[target.'cfg(target_env = "msvc")'.build-dependencies]
+cc = { version = "1.0" }
```

Comment:
We should not depend on `cc` if no 'bundled*' feature is enabled, no ?

Reply:
You're right, I could actually avoid it. I've pushed a fix